### PR TITLE
[Do not merge] Created a separate script to do release builds

### PR DESCRIPTION
**Pull request by LalatenduMohanty**
Signed-off-by: Lalatendu Mohanty lmohanty@redhat.com

### CODE REVIEW

**Comment by LalatenduMohanty** [comment]
I am getting following error while doing the build [1] 
"GenericError: Invalid SCM URL: https://github.com/projectatomic/adb-atomic-developer-bundle/blob/master/build_tools/kickstarts/centos-7-adb-vagrant.ks#a8d6dd0fda693b3ade75d3425236282c7d3ba5aa"

[1] https://cbs.centos.org/koji/taskinfo?taskID=58780

File: build_tools/build_scripts/do_vagrant_cbs_release.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+#To see all options in koji -p command check "koji -p cbs image-build --help"
+COMMIT_ID=`git log | head -n 1 | cut -f 2 -d " "`
+
+koji -p cbs image-build \
+  centos-7-adb 1.5 atomic7-adb-common-el7 \
+  http://mirror.centos.org/centos/7/os/x86_64/ x86_64 \
+  --release=1 \
+  --distro RHEL-7.0 \
+  --ksver RHEL7 \
+  --ksurl=https://github.com/projectatomic/adb-atomic-developer-bundle/blob/master/build_tools/kickstarts/centos-7-adb-vagrant.ks#$COMMIT_ID \
```
Comment (LalatenduMohanty) [inline build_tools/build_scripts/do_vagrant_cbs_release.sh:11]: Need to provide a valid SCM url

**Comment by praveenkumar** [comment]
@LalatenduMohanty Anything blocking it?

**Comment by LalatenduMohanty** [comment]
@praveenkumar yes. It is blocked till we can use git.centos.org

**Comment by LalatenduMohanty** [comment]
There is no progress in CentOS SIGs with respect to git.centos.org. Hence closing the PR for now

**Pull request closed without merging**